docs/tomcat-docker-tutorial/README.md

```diff
@@ -0,0 +1,89 @@
+# Collecting JMX metrics from a Tomcat v9.0.8 running inside docker container
+
+## Prerequisites
+
+##  1. <a name='InstalltheInfrastructureagent'></a>Install the Infrastructure agent and JMX integration
+
+- [Install Infrastructure for Linux using the package manager](https://docs.newrelic.com/docs/infrastructure/install-configure-manage-infrastructure/linux-installation/install-infrastructure-linux-using-package-manager)
+
+  or 
+
+- [Install Infrastructure for Windows Server using the MSI installer](https://docs.newrelic.com/docs/infrastructure/install-configure-manage-infrastructure/windows-installation/install-infrastructure-windows-server-using-msi-installer)
+
+- [Install New Relic JMX integration](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/jmx-monitoring-integration#install)
+
+## 2. Expose JMX from Tomcat
+
+For this tutorial we will run a Tomcat in inside Docker using the following Dockerfile:
+
+```bash
+
+FROM tomcat:9.0.8
+
+ARG CATALINA_OPTS
+ENV CATALINA_OPTS="-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.local.only=false -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.port=9010 -Dcom.sun.management.jmxremote.rmi.port=9010 -Djava.rmi.server.hostname=0.0.0.0 -Dcom.sun.management.jmxremote.ssl=false"
+
+EXPOSE 9010
+EXPOSE 8080
+```
+
+Build and run the image, exposing the JMX port 9010:
+
+```bash
+docker build -t tomcat_908_jmx . && docker run -d -p 9010:9010 --name=tomcat_908_jmx tomcat_908_jmx
+```
+
+Test the JMX connection:
+
+```bash
+echo '*:*' | nrjmx -port 9010 -host $(docker inspect --format '{{ .NetworkSettings.IPAddress }}' tomcat_908_jmx)
+```
+
+##  3. Configure JMX integration
+
+### 3.1 First step is creating a JMX integration configuration file `/etc/newrelic-infra/integrations.d/jmx-config.yml`
+
+for the jmx_host use this command to obtain ip address of the container `docker inspect --format '{{ .NetworkSettings.IPAddress }}' tomcat_908_jmx` (binding to 0.0.0.0 not working in this case)
+
+```yaml
+integration_name: com.newrelic.jmx
```

Comment:
this is old config format, I think we should use the "newer" (>1y old) one https://docs.newrelic.com/docs/create-integrations/infrastructure-integrations-sdk/specifications/host-integrations-newer-configuration-format